source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,168 @@
 
-## HTML5
+## Browser support
 
-* Always quote attribute values - the decision tree for remember when to
-  selectively quote is too much mental overhead, so just quote them
-* Always use well-formed HTML - implicitly closed elements such as
-  `<p>` cause unnecessary parsing overhead and can cause issues in
-  accessibility
+The GOV.UK Service Manual lists [browsers that we are required to support](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in). Specific
+services, such as GOV.UK Frontend, may have [stricter standards of browser
+support](https://github.com/alphagov/govuk-frontend#browser-support).
 
-## Use [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) roles appropriately
+Whatever your level of browser support, HTML should be written in such a way
+that older browsers see an appropriate fallback or warning message. For example,
+some browsers do not support the `<video>` element, so you should ensure that
+some contextual information is included (the `<p>` element in this example):
 
-### Guidance on using [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmarks
+```html
+<video width="320" height="240" controls>
+  <source src="movie.mp4" type="video/mp4">
+  <source src="movie.ogg" type="video/ogg">
+  <p>Unable to load video. Please upgrade your browser.</p>
+</video>
+```
 
-When you use HTML5 sectioning elements it's a good idea to include
-the appropriate [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) landmark roles as well. The trick when you're
-using both HTML5 and [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) is to put the landmark on the sectioning
-element itself because this avoids the information being duplicated by
-screen readers that support both technologies.
+When using structual HTML5 elements, such as `<main>`, ensure that you include
+a shim that allows them to be styled in legacy browsers:
 
-Example:
+```html
+<!--[if lt IE 9]><script src="https://assets.publishing.service.gov.uk/static/ie-a4524544a53d57a7e259b4bb966b9c32557c98c920b77e52d09304642b68401a.js" crossorigin="anonymous" integrity="sha256-pFJFRKU9V6fiWbS7lmucMlV8mMkgt35S0JMEZCtoQBo="></script><![endif]-->
+```
+
+Your page should work with just HTML, before adding anything else like CSS and
+JavaScript. Read [building a resilient frontend using progressive enhancement](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).
+
+## Element structure
+
+We don't prescribe whether to use single or double quotes in attributes,
+whether to omit values from attributes that don't need them, or whether
+to add [trailing end slashes to void elements](https://www.w3.org/TR/html5/syntax.html#start-tags).
 
-    <header role="banner"></header>
+What's important is that the approach you adopt is consistent throughout
+your repository, or at the very least consistent throughout your file.
```

Comment:
This is much better and I appreciate you taking the time to rewrite. However, I'm left wondering if this section is needed at all? Thoughts?

Reply:
Fair point. I've removed the section and condensed this all into a single paragraph and slipped it into the `Document structure` section. Think it's worth noting for avoidance of doubt.